src/main/resources/com/parasoft/findings/jenkins/coverage/cobertura.xsl

```diff
@@ -1,12 +1,59 @@
 <?xml version="1.0" encoding="UTF-8"  standalone="yes"?>
 <xsl:stylesheet version="3.0"
-    xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
-    xmlns:xs="http://www.w3.org/2001/XMLSchema"
-    xmlns:map="http://www.w3.org/2005/xpath-functions/map">
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:map="http://www.w3.org/2005/xpath-functions/map">
     <xsl:variable name="toolName" select="/Coverage/@toolId"/>
     <xsl:variable name="toolVer" select="/Coverage/@toolVer"/>
     <xsl:variable name="toolDispName" select="/Coverage/@toolDispName"/>
     <xsl:param name="pipelineBuildWorkingDirectory"><xsl:value-of select="/Coverage/@pipelineBuildWorkingDirectory"/></xsl:param>
+
+    <xsl:template name="getUriWithoutFilePrefix">
+        <xsl:param name="rawUri"/>
+        <xsl:choose>
```

Comment:
I haven't carefully examined the specific logic in the regular expressions, and I'm curious if it's possible to combine the three regular expressions.